src/lib/components/dateSelector/index.tsx

```diff
@@ -12,8 +14,9 @@ import styles from './style.module.scss';
 import './datepicker.scss';
 import calendarIcon from './icons/calendar.svg';
 
-dayjs.extend(localeData);
+export type supportedDayJSLocales = 'en' | 'de';
```

Comment:
Not important, just adding my opinion - IMO this name can be more concise by saving `supported`, because I feel like `supported` is already implied by enum, limiting it to a certain set of values.

Reply:
To be honest, I wanted it to be a bit more verbose so that in the future, when someone tries using this component inside the other repos and stumbles across this property it provides more context. `dayJSLocales` for example wouldn't be able to convey the whole story as it is not an enum of all the dayjs locales but only a subset of those, the ones that are supported by the component (because they were imported). Even if you'd see that there's only 2 entries currently in the union type, you could come to the conclusion that the type itself might just not be up to date and try to pass in another of dayjs' locales.
But yeah with names there's probably never a 100% correct answer and you can find good arguments for either side really. But in my experience it rarely hurts to be a bit "too verbose".

Reply:
> But in my experience it rarely hurts to be a bit "too verbose".
Yes, agreed!